data/assets/scene/solarsystem/missions/voyager/voyager1.asset

```diff
@@ -95,8 +95,9 @@ local VoyagerTrailCruiseEarthJupiter = {
     Color = { 0.70, 0.50, 0.20 },
     StartTime = "1977 SEP 05",
     EndTime = "1979 MAR 04",
-    -- 545 is the number of days between the Start and End time
-    SampleInterval = 545 * 2
+    -- 1 sample every 2d
+    SampleInterval = openspace.duration("1977 SEP 05", "1979 MAR 04") /
```

Comment:
Maybe we should store the dates in local variables and minimize the risk of errors.

Reply:
I opted against that since it would lead to a lot of variables at the top of the file that would hurt readability